assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
+				type="text" placeholder="code postal" pattern="\d{5}" title="Entrez le code postal de l'établissement où le salarié sera employé">
```

Comment:
Ah right, those damn arrows on desktop browsers… I keep on forgetting them, they are more a problem than an improvement, really  :-/

Have you tried `postal-code`? I think it should be unsupported and thus default to `text` on most browsers, but we would be forward-compatible, and it is quite probable autocompleters already take advantage of this.